src/components/AuthModal/WalletList/index.tsx

```diff
@@ -37,6 +39,7 @@ export const WalletList: React.FC<WalletListProps> = ({ onLogin }) => {
 
   return (
     <div css={styles.container}>
+      <NoticeInfo description={<Trans i18nKey="authModal.binanceChainWalletMigrationInfo" />} />
```

Comment:
I'd add a bottom margin to space the notice out of the list of wallets. Perhaps something like 16px.